app/views/tutorials/analyzing.pt.html.erb

```diff
@@ -0,0 +1,93 @@
+<h1>Tutorial</h1>
+<h2>Analisando um repositório</h2>
+
+<hr>
+
+<h3>Requisitos</h3>
+
+A fim de criar um projeto, você deve:
+<ul>
+  <li><%= link_to(t('signed_up'), new_user_registration_path) %> na plataforma</li>
+  <li>Estar <%= link_to(t('signed_in'), new_user_session_path) %></li>
+  <li>E ter uma URL do repositório que você deseja analisar</li>
+</ul>
+
+<hr>
+
+<h3>Passos</h3>
+
+<h4>Criando um Projeto</h4>
+
+<ol>
+  <li>Clique em "Projeto" no menu superior</li>
+  <li>Clique em "Novo Projeto"</li>
+  <li>Preencha o formulário com um "Nome", que deve ser único</li>
+  <li>Preencha o formulário com uma "Descrição"</li>
+  <li>Clique em "Salvar". Agora você deve estar na página do projeto, parabéns!</li>
+</ol>
+
+<h4>Criando um Repositório</h4>
+
+<ol>
+  <li>A partir da página do projeto, clique em "Novo Repositório"</li>
+  <li>Preencha o formulário, garantindo que:
+    <ul>
+      <li>O "Nome" deve ser único</li>
+      <li> Você deve escolher uma "Licença" Open Source na plataforma de acordo com o seu código</li>
+      <li>O campo "Tipo" está relacionado com a ferramenta de controle de versão do código (Git, por exemplo)</li>
+      <li>Preencha o "Endereço" com a URL a partir da qual podemos fazer o download do código</li>
+      <li>Se você quiser analisar seu projeto periodicamente, configure-o (você pode também executá-lo a partir da interface sempre que quiser)</li>
+      <li>Por fim, a "Configuração" representa o conjunto de métricas e interpretações que vamos usar para analisar o seu projeto</li>
+    </ul>
```

Comment:
"O "Nome" deve ser único" -> "O "Nome" SEJA único".
There are other errors like this within the ul tag.